docs/content/contributing/community/office-hours.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Office Hours"
+date: 2018-09-28T10:40:50-07:00
```

Comment:
Do we want the date to be consistent with the old developers-meeting file? As it's a new topic, we may want to consider updating